Client/Frontend/Login Management/LoginListViewController.swift

```diff
@@ -192,9 +192,14 @@ class LoginListViewController: SensitiveViewController {
         selectionButton.setTitleColor(UIColor.theme.tableView.rowBackground, for: [])
         selectionButton.backgroundColor = UIColor.theme.general.highlightBlue
 
-        let theme = BuiltinThemeName(rawValue: ThemeManager.instance.current.name) ?? .normal
-        if theme == .dark {
-            searchController.searchBar.barStyle = .black
+         if let textFieldInsideSearchBar = self.searchController.searchBar.value(forKey: "searchField") as? UITextField,
```

Comment:
Hi @KrystynaKruchkovska, PR looks good although I would suggest suggest a more cleaner approach so we can avoid KVC and use something like below to get the

**Getting text field:** 
`let searchTextField = searchController.searchBar.searchTextField`

**Updating the attributes: (both approaches are fine)** 
`searchTextField.defaultTextAttributes = [NSAttributedString.Key.foregroundColor:UIColor.black]`

or 

`searchTextField.defaultTextAttributes[NSAttributedString.Key.foregroundColor] = UIColor.black`

**Sample final result** 
```
        let isDarkTheme = ThemeManager.instance.currentName == .dark
        let searchTextField = searchController.searchBar.searchTextField
        if isDarkTheme {
            searchTextField.defaultTextAttributes[NSAttributedString.Key.foregroundColor] = UIColor.white
        } else {
            searchTextField.defaultTextAttributes[NSAttributedString.Key.foregroundColor] = UIColor.black
        }
    
        if let glassIconView = searchTextField.leftView as? UIImageView {
            //Magnifying glass
            glassIconView.image = glassIconView.image?.withRenderingMode(.alwaysTemplate)
            glassIconView.tintColor = UIColor.theme.tableView.headerTextLight
        }
```